internal_worker_base.go

```diff
@@ -214,7 +218,8 @@ func (bw *baseWorker) pollTask() {
 	var err error
 	var task interface{}
 	bw.retrier.Throttle()
-	if bw.pollRateLimiter.Consume(1, time.Millisecond*100) {
+	if bw.pollLimiter.Wait(bw.limiterContext) == nil {
```

Comment:
ah.. the diff viewer made me think the else{} was for the if {rateLimiter.Wait()} - nvm